### PR TITLE
fix(peerDependencies): version bump

### PR DESCRIPTION
**Pull request by wildpow**
NPM @7 install can not resolve dependencies in React 17 project. React 16.3.0 -> 17.0.0 fixes this issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "pretty-docs": "prettier-eslint docs/src/**/*.js --write --no-semi --single-quote --trailing-comma=all --print-width 80"
   },
   "peerDependencies": {
-    "react": "^16.3.0"
+    "react": "^17.0.0"
```
Comment (andtos90) [inline package.json:53]:
```suggestion
    "react": ">= 16.3.0 < 18"
```

**Comment by ChihweiLHBird** [comment]
Any update on this PR? I really hope we can use it with React 17 and npm 7.

**Comment by sghall** [comment]
Hey there.  @wildpow  thanks for putting this up.  If I could see the tests and the docs run against react 17 and it looked good, I would just merge this and release it.  I just tried updating the deps to do that and, as you can imagine, that is much more complicated than just changing the peer range.  I was able to solve the Enzyme issues but there's more stuff that needs to be done.  I wanted to let you know I saw this.  If you wanted to add this stuff to the PR I would pull it and look at it.  I know that is unlikely.   I will look at this again tomorrow but this is unlikely to get resolved until the weekend.

**Comment by sghall** [comment]
The updated peer range was released in version `6.5.0` please install the latest...

**Pull request closed without merging**